windows/RNSVG/RenderableView.cpp

```diff
@@ -96,40 +95,16 @@ void RenderableView::UpdateProperties(IJSValueReader const &reader, bool forceUp
         if (propertyValue.IsNull()) {
           m_strokeLineCap = parent.StrokeLineCap();
         } else {
-          auto const &strokeLineCap{propertyValue.AsInt32()};
-          switch (strokeLineCap) {
-            case 2:
-              m_strokeLineCap = Geometry::CanvasCapStyle::Square;
-              break;
-            case 1:
-              m_strokeLineCap = Geometry::CanvasCapStyle::Round;
-              break;
-            case 0:
-            default:
-              m_strokeLineCap = Geometry::CanvasCapStyle::Flat;
-              break;
-          }
+          m_strokeLineCap = propertyValue.AsInt32();
```

Comment:
Is it appropriate here to ensure the number value is within the expected valid range? Do the input validation here or later?

Reply:
It's an enum. If it's out of range, the converter will use the default.